GDJS/Runtime/types/project-data.d.ts

```diff
@@ -183,6 +294,14 @@ declare interface LayoutNetworkSyncData {
   extVar?: {
     [extensionName: string]: VariableNetworkSyncData[];
   };
+  time?: TimeManagerSyncData;
+  tween?: TweenManagerNetworkSyncData;
```

Comment:
(here same: it's convenient, but it's actually the responsibility of the tween extension to "hijack this data to add some stuff" but the game engine should not be aware of it in theory)